apps/__init__.py

```diff
@@ -1,3 +1,5 @@
+import importlib
 import logging
 
 logger = logging.getLogger(__name__)
+celery = getattr(importlib.import_module("adhocracy-plus.config.celery"), "celery")
```

Comment:
So we can move this part to `adhocrac-plusy/__init__.py` and do:
```
from .config.celery import app as celery_app

__all__ = ('celery_app',)
```
then make use of `from celery import shared_task` decorator instead of importing the configured app. then we don't have issues with the hyphen.
ref: https://docs.celeryq.dev/en/stable/django/first-steps-with-django.html#using-celery-with-django

Reply:
done